### ISSUE TITLE
O3653-7 Compliance review: need alt text for figures

### ISSUE THREAD

**Issue by beckandros** [OfficeDev/hands-on-labs#43]
Module Number:  O3653-7 
Module Title:  Deep Dive into the Office 365 APIs for OneNote services 
Path of file: https://github.com/OfficeDev/trainingcontent-NDA/blob/master/O3653/O3653-7%20Deep%20Dive%20into%20the%20Office%20365%20APIs%20for%20OneNote%20services/Lab.md
### Details of issue

Images lack alt text. Please provide.
